src/util/worker/future.rs

```diff
@@ -52,12 +53,15 @@ pub trait Runnable<T: Display> {
 pub struct Scheduler<T> {
     name: Arc<String>,
     sender: UnboundedSender<Option<T>>,
+    metrics_pending_count: Gauge,
```

Comment:
Maybe s/`pending_count`/`pending_task_count` and add some comment to tell this is a local copy?

Reply:
emmm.. "local copy" is not very accurate as well. It is a shared global variable, with labels carried.